fedimintd/src/lib.rs

```diff
@@ -67,3 +69,21 @@ pub fn attach_default_module_gen_params(
             },
         );
 }
+
+pub fn default_esplora_server(network: Network) -> BitcoinRpcConfig {
+    let url = match network {
+        Network::Bitcoin => Url::parse("https://blockstream.info/api/")
+            .expect("Failed to parse default esplora server"),
+        Network::Testnet => Url::parse("https://blockstream.info/testnet/api/")
+            .expect("Failed to parse default esplora server"),
+        Network::Regtest => {
+            Url::parse("http://127.0.0.1:50002/").expect("Failed to parse default esplora server")
+        }
+        Network::Signet => Url::parse("https://mutinynet.com/api/")
```

Comment:
Just curious, is our plan to mainly only support mutiny net (as opposed to the non-forked "other" signet) ?

Reply:
No, but it's also just a default for the config gen parameter, i.e. this should be changeable via the setup UI eventually and is already changeable in the consensus config after this PR (with some annoying manual steps).

Reply:
I wish rust-bitcoin had a way to specify a custom signet here

Reply:
I thought about opening a PR, but iirc there wouldn't be a good way to inject all the network-specific constants, so this would mean a major API and functionality change :/